common-lb/src/main/java/com/rtbhouse/grpc/loadbalancer/UpdatableServerList.java

```diff
@@ -0,0 +1,16 @@
+package com.rtbhouse.grpc.loadbalancer;
+
+public interface UpdatableServerList {
```

Comment:
Czy to służy do utrzymywania listy aktualnie dostępnych serwerów? Jeśli tak to bardziej znacząca nazwa by się przydała.

Reply:
Hmm, ogólnie sprawa wygląda tak: do przechowywania listy aktualnie dostępnych serwerów służy DirectServerList, z pakietu loadbalancer-standalone. Ono implementuje ten interfejs. Za to ten interfejs powstał po to, żeby można było współdzielić kod między iteracjami, podmieniać implementację listy serwerów jako np. lista z zookepera czy lista w pamięci (DirectServerList).

Ale po drodze okazało się, że trochę nam się zmienił model komunikacji między klasami i nie wykorzystujemy tego tak jak zakładaliśmy; usuwam więc ten interfejs i zmieniam nazwę DirectServerList na AvailableServerList.